polygeist/tools/cgeist/Test/Verification/booleanconversion.c

```diff
@@ -0,0 +1,72 @@
+// RUN: cgeist %s --function=* -S -O0 | FileCheck %s
+
+#include <stdbool.h>
+
+// CHECK-LABEL:   func.func @int_conversion(
+// CHECK-SAME:                              %[[VAL_0:.*]]: i32) -> i8
```

Comment:
Noting that the return value is `bool` which should be encoded in LLVM IR as an i1 not an i8.

Reply:
:+1: